packages/x-components/src/x-modules/search/store/__tests__/actions.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('testing search module actions', () => {
 
```

Comment:
I think the `mockedEmptySearchResponse` is not really a mock. It is not replacing a real "thing" and acting like it (As the MockedAdapter).

To me is another stub. I think we should create a `getEmptySearchResponseStub` along the other stub factories, and use here like any other stub